### ISSUE TITLE
Issue with "wrongly-placed shaded zones?"

### ISSUE THREAD

**Issue by DarrenMcRoy** [zooniverse/Penguins#70]
I'm seeing reports on Talk that a large number of images are being served with the shaded zones being placed "in the foreground or on the sides of the site." I don't fully understand, but it seems to be causing a lot of concern and inability to mark properly?

http://talk.penguinwatch.org/#/boards/BPZ0000001/discussions/DPZ0000c2c?page=3&comment_id=554a5765a77a1b126200000c

Zuzi tells me that she spoke with Tom and Caitlin yesterday about this, and Tom said it sounded like a major concern.

Is anyone looking into this? Does anyone understand the issue better than I do?

**Comment by penguintom79** [comment]
Yes, forgot to mention this to Cam – Cam those images we gave you don’t need any shaded areas, and I think for a couple of cameras the view might have changed slightly.

Can you see which images? We can do new ROIs.

Cheers,
Tom


Dr Tom Hart, Penguinologist.
Department of Zoology,
South Parks Road,
Oxford,
OX1 3PS.

+44 1865 600 170

www.penguinwatch.org<http://www.penguinwatch.org/>
www.penguinlifelines.org<http://www.penguinlifelines.org/>

**Comment by caitlinblack** [comment]
Hi Darren,
I think there is a problem with the ROI. All of the images that she flagged are from one site- Booth Island. The camera is named BOOTb. Is there an easy way to change the ROI to the whole image?

Cheers,
Caitlin Black

DPhil Student
Ocean Research and Conservation Group

University of Oxford
Department of Zoology
South Parks Rd.
Oxford OX1 3PS
UK

+44 (0) 1865 281 329
www.penguinlifelines.org
www.penguinwatch.org

**Comment by DarrenMcRoy** [comment]
@camallen can you work with Tom and Caitlin on this?

If you're looking for examples of images that are having this issue, there are a ton provided by users here: http://talk.penguinwatch.org/#/boards/BPZ0000001/discussions/DPZ0000c2c?page=3&comment_id=554a5765a77a1b126200000c

Thanks!!

**Comment by camallen** [comment]
Sso you can see the general shape of the existing ROI's here, http://www.penguinwatch.org/rois.html and this is the file we need to fix the coords for https://github.com/zooniverse/Penguins/blob/master/public/roi.tsv

I assume that the camera moved between seasons? If so will, which sites have moved? Also do new sites need ROI filters?

**Comment by caitlinblack** [comment]
Hey Cam,
I don’t think any of the sites need the new filters. Is it possible to make BOOTb2015 undefined for now? We will look and see if any ROIs have shifted from last year, but let’s assume they haven’t. The one exception is LOCKa (the one you asked about earlier), which has definitely moved, but can be set as undefined.

Hope that helps,
Caitlin Black

DPhil Student
Ocean Research and Conservation Group

University of Oxford
Department of Zoology
South Parks Rd.
Oxford OX1 3PS
UK

+44 (0) 1865 281 329
www.penguinlifelines.org
www.penguinwatch.org

**Comment by camallen** [comment]
@penguintom79 said:
>  Yes, forgot to mention this to Cam – Cam those images we gave you don’t need any shaded areas, and I think for a couple of cameras the view might have changed slightly.

@caitlinblack said:
>I don’t think any of the sites need the new filters. Is it possible to make BOOTb2015 undefined for now? We will look and see if any ROIs have shifted from last year, but let’s assume they haven’t. The one exception is LOCKa (the one you asked about earlier), which has definitely moved, but can be set as undefined.

Can you verify all the site filenames you gave me do not correlate to any of these roi names [here](http://www.penguinwatch.org/rois.html)? If none match then they will not have roi filters applies to them. E.g. image [APZ00049me](https://api.zooniverse.org/projects/penguin/subjects/APZ00049me) had the filename "BOOTb2015/**BOOTb2014b**_001620.JPG" which matched `BOOTb2014b` roi.

I fixed the BOOTb2014b image to have no ROI as per the closed ticket above. Can you verify which LOCKa site in the ROI's you want to remove?
